service/arm_templates/node_setup.json

```diff
@@ -71,6 +78,16 @@
     },
     "loadBalancerBackendPoolIDs": {
       "type": "array"
+    },
+    "assignContributorRole": {
+      "type": "bool"
+    },
+    "contributorRoleDefinitionId": {
+      "type": "string",
+      "defaultValue": "b24988ac-6180-42a0-ab88-20f7382dd24c",
```

Comment:
This seems to be the GUID that is always attributed to the contributor role, no matter the account/subscription/resource group. 

Take a look here: https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-manager-api-authentication#get-azure-rbac-role-identifier

Reply:
OMG Couldn't that be a `Contributor`...

Reply:
I wish, that feels so dirty..